src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -264,13 +285,29 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     })()
   }, [accountId, assetId, chainId, fromAddress])
 
+  const memo = useMemo(() => {
+    if (isRunePool && state) {
+      const balanceCryptoPrecision = bnOrZero(amountAvailableCryptoPrecision.toPrecision())
```

Comment:
suggestion: We shouldn't need to `toPrecision()` this guy?
```suggestion
      const balanceCryptoPrecision = bnOrZero(amountAvailableCryptoPrecision)
```

Reply:
Of course!, fixed in [1f9d825](https://github.com/shapeshift/web/pull/7449/commits/1f9d8254294f6843bde971347039e344909ac032)